config/initializers/docker.rb

```diff
@@ -17,4 +17,16 @@
     warn "Unable to connect to docker!"
     Airbrake.notify($!)
   end
+
+  # ensure that --cache-from is supported (v13+)
+  begin
+    Timeout.timeout(1) do
+      if Integer(`docker -v`[/Docker version (\d+)/, 1]) < 13
```

Comment:
I think that `docker -v` just returns the version of the client binary.  Which isn't necessarily the same as the docker daemon you're connecting to.

If you run `docker version`, you get info about both the client and the server.

```
jmoter@deploy1:~$ docker version
Client:
 Version:      17.04.0-ce
 API version:  1.28
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 17:54:19 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.04.0-ce
 API version:  1.28 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 18:01:50 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

Reply:
Looks like the `Docker.version` method of the docker-api gem returns the version of the server.

https://github.com/swipely/docker-api#global-calls